packages/common/src/types.ts

```diff
@@ -745,6 +747,22 @@ export interface WalletExportFormat {
     };
 }
 
+export function parseWallet(walletString: JsonString): WalletExportFormat {
```

Comment:
I see you have documented it, but is there a reason for not supporting both formats?

The genesis format is relevant for people using company identities, and that is also supported by both Rust and .NET SDKs.

Reply:
I agree it would be nice to add something that parses the genesis format as well. It is however already supported for generating an `AccountSigner`, so you can use it, we just don't have a function that validates the format.